### PR TITLE
Bug 1945594: fixing-project-creation-and-updated-packageJson

### PR DESCRIPTION
**Pull request by makambalaji**
**problem**: Project creation feature flakiness
**Solution**: Fixed the issue and updated package.json

**Note**: This PR is Manual cherrypick

image

image

### CODE REVIEW

**Review by gajanan-more — APPROVED** [review]
Screenshot from 2021-04-01 15-17-01


Working as expected!!

/lgtm

File: frontend/packages/dev-console/integration-tests/features/common/project-creation.feature
Comment (andrewballantyne) [inline frontend/packages/dev-console/integration-tests/features/common/project-creation.feature:10]: This an indentation problem?

Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/common/project-creation.feature:10]: nope, this is expected. Based on vs code plugin "Siarhei Kuchuk". This indentation appears

Comment (andrewballantyne) [reply frontend/packages/dev-console/integration-tests/features/common/project-creation.feature:10]: Most of your other `.feature` files don't have this indentation, seems odd... but hey, right now QE is the only one maintaining these... if you're good with the indents, who am I to say it's odd 🙂

**Review by andrewballantyne — APPROVED** [review]

**Comment by andrewballantyne** [comment]
/hold

Kubevirt-plugin is broken. See [the slack thread](https://coreos.slack.com/archives/C6A3NV5J9/p1618195395496000) for more information.

**Comment by andrewballantyne** [comment]
Fix being applied here: https://github.com/openshift/console/pull/8619

@makambalaji hopefully it'll pass once this merges.

**Comment by mfojtik** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.


* Score: *0.20*
* Reason: *maximum total for this z-stream is 10*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Comment by makambalaji** [comment]
#8619 got merged.
/hold cancel

**Pull request merged**
